include/libtorrent/peer_info.hpp

```diff
@@ -416,7 +416,7 @@ namespace libtorrent
 	};
 
 	// internal
-	struct TORRENT_EXPORT peer_list_entry
+	struct TORRENT_EXTRA_EXPORT peer_list_entry
```

Comment:
I mostly agree with this. I added this a long time ago for research purposes. I wonder if there's any other way of accessing the peers. maybe in resume data. I think the resume data limits the number of peers being saved though